metadata-ingestion/tests/integration/dremio/dremio_mces_golden.json

```diff
@@ -0,0 +1,2856 @@
+                    "type": "COPY"
+                }
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1697353200000,
+        "runId": "dremio-2023_10_15-07_00_00",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:dremio,dremio.s3.warehouse.sample.parquet,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "datasetProperties",
+    "aspect": {
+        "json": {
+            "customProperties": {},
+            "externalUrl": "http://localhost:9047/api/v3/source/\"s3\"/\"warehouse\".\"sample.parquet\"",
+            "name": "sample.parquet",
+            "qualifiedName": "s3.warehouse.sample.parquet",
+            "description": "",
+            "created": {
+                "time": 0
+            },
+            "tags": []
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1697353200000,
+        "runId": "dremio-2023_10_15-07_00_00",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:dremio,dremio.s3.warehouse.sample.parquet,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "subTypes",
+    "aspect": {
+        "json": {
+            "typeNames": [
+                "Table"
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1697353200000,
+        "runId": "dremio-2023_10_15-07_00_00",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:dremio,dremio.s3.warehouse.sample.parquet,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "dataPlatformInstance",
+    "aspect": {
+        "json": {
+            "platform": "urn:li:dataPlatform:dremio"
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1697353200000,
+        "runId": "dremio-2023_10_15-07_00_00",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:dremio,dremio.s3.warehouse.sample.parquet,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "browsePathsV2",
+    "aspect": {
+        "json": {
+            "path": [
+                {
+                    "id": "s3",
+                    "urn": "urn:li:container:63a316133b08a091e919dc8c7a828a4d"
+                },
+                {
+                    "id": "warehouse",
+                    "urn": "urn:li:container:b6e7d0c364ccc53ccb02b438999fda9a"
+                }
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1697353200000,
+        "runId": "dremio-2023_10_15-07_00_00",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:dremio,dremio.s3.warehouse.sample.parquet,PROD)",
```

Comment:
Is your question related to why we have `urn:li:dataset:(urn:li:dataPlatform:dremio,dremio.my_space.my_folder.raw,PROD)` and not `urn:li:dataset:(urn:li:dataPlatform:dremio.my_space.my_folder.raw,PROD)`? If so then this is because Dremio effectively has a single database under the hood called "DREMIO". It is not gathered from any of the system tables but all ODBC and JDBC connections do see it. Therefore, to get things like Tableau working correctly the DB needs to be added to the URN for lineage.

Reply:
Yes that's the question. Probably tableau-like sources should ignore dremio prefix it its just a constant. WDYT ?

I think this might just be an inner adapter layer detail (similar to how the database name is always `awsdatacatalog` for athena ) and we should ignore it. cc: @hsheth2 to weigh in .

Reply:
I think it's fine to leave as-is, assuming that this makes other BI tools work out of the box